common/chroot-style/ethereal.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/sh
+#
+# This chroot script uses symlinks to emulate being in a chroot,
+# it also symlinks the root directory to the masterdir and assumes
+# base-chroot is installed in the host system.
+#
+# It will damage your host system, only use it in disposable
+# containers.
+#
+# Note that this assumes the following.
+# The xbps-src git checkout not is mounted at /void-packages
+# There is a symlink from / to /void-packages/masterdir
+#
+
+readonly MASTERDIR="$1"
+readonly DISTDIR="$2"
+readonly HOSTDIR="$3"
+readonly EXTRA_ARGS="$4"
+readonly CMD="$5"
+shift 5
+
+if [ -z "$MASTERDIR" -o -z "$DISTDIR" ]; then
+	echo "$0 MASTERDIR/DISTDIR not set"
```

Comment:
Perhaps a slightly more helpful message here?  Something about linking the root.

Reply:
That is an error for all chroot-styles, it means xbps-src somehow passed something wrong.